posts/2021-01-06-ocp-log-forwarding.adoc

```diff
@@ -324,9 +324,19 @@ Proceed to the `Review` section and submit. Copy the generated token value to us
 +
 
 
-== Integrating Open Liberty JSON Logs
+== Integrating Open Liberty Logs
 
-Since Open Liberty application pods output logs in JSON format, you must enable Fluentd to parse the JSON fields from the message body.
+Open Liberty has the capability to emit logs to standard out, or in Liberty terms _console output_, in either _JSON_ or _simple_ (plain text) format. These are the logs that the OpenShift platform consumes and redirects to configured destinations. When emitting logs in simple format only message logs are included. The intention is that the output is for human consumption. With JSON formatted output you can configure any type of log to output such as message logs, trace logs, ffdc logs, access logs and audit logs.  The intention is to consolidate all the logging data into a single stream in a mapped format. This logging strategy is optimized for a containerized environment. Ultimately this allows for the log data to be easily indexed and queried by monitoring tools down stream. For example, the JSON data can be leveraged to create meaningful visualizations on log analysis dashboards. The Open Liberty team has already created publicly available dashboards for Kibana and Splunk, but you can modify or create your own dashboards to your monitoring needs. Note that you are still able to use Kibana or Splunk without JSON logging, but without the benefit of being able to leverage all log types and data and you will be limited in terms of querying capabilities.
+
+For more information regarding Open Liberty logging see Open Liberty's link:https://openliberty.io/docs/20.0.0.12/log-trace-configuration.html[logging and tracing documentation]. For specifics regarding JSON logging, see the link:https://openliberty.io/docs/20.0.0.12/log-trace-configuration.html#json[Open Liberty JSON logging documentation]. Also see the link:https://openliberty.io/docs/20.0.0.12/log-trace-configuration.html#json[JSON log events reference list] for a complete index of available fields from the different log types.
```

Comment:
suggested edit- note that you can specify `latest` in OL docs URLs and the doc will default to the most recent version.

For more information about Open Liberty logging, see the Open Liberty link:https://openliberty.io/docs/latest/log-trace-configuration.html[logging and tracing documentation]. For specifics regarding JSON logging, see the link:https://openliberty.io/docs/latest/log-trace-configuration.html#json[Open Liberty JSON logging documentation]. Also, see the link:https://openliberty.io/docs/20.0.0.12/log-trace-configuration.html#json[JSON log events reference list] for a complete index of available fields for the different log types.